examples/experimental/ProductionMonitoring/stream_table_api.ipynb

```diff
@@ -0,0 +1,170 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "eccb44d3",
+   "metadata": {},
+   "source": [
+    "# Weave StreamTable Usage\n",
+    "\n",
+    "This notebook demonstrates basic Weave StreamTable usage with interactive examples.\n",
+    "\n",
+    "## Step 0: Setup\n",
+    "\n",
+    "All the StreamTables created in this notebook will be saved to the WB_PROJECT under the WB_ENTITY account on the public W&B cloud. \n",
+    "\n",
+    "**Please login to W&B and set your WB_ENTITY** before running this demo."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "51c30e7a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import weave\n",
+    "from weave.monitoring import StreamTable"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5684f0b8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "WB_ENTITY = \n",
+    "WB_PROJECT = \"mesa\"\n",
+    "STREAM_TABLE_NAME = \"my_stream_table\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6f676d96",
+   "metadata": {},
+   "source": [
+    "## Step 1: Define a StreamTable\n",
+    "\n",
+    "StreamTable has a single required argument: the name of the StreamTable object.\n",
+    "\n",
+    "```python\n",
+    "st = StreamTable(\"stacey/mesa/my_stream_table\")\n",
+    "```\n",
+    "\n",
+    "This takes the form `my_wb_entity/my_wb_project_name/my_stream_table_name` where you can modify the component names to the relevant strings (e.g. your W&B username or shared W&B team name, a new or existing W&B project name).\n",
```

Comment:
I think `entity` is optional right? might be worth mentioning that?

Reply:
Yes, thank you, this is one of my top TODOs—it turns out to be optional in practice, but the actual StreamTable code has a comment that says the following—and then comments out the check :))))—so I wasn't sure if we should leave the soft requirement in for authentication reasons, esp if/when folks are running this notebook for the first time/on a new machine or something.

"# For now, we force the user to specify the entity and project
        # technically, we could infer the entity from the API key, but
        # that tends to confuse users.
        # if entity_name is None or entity_name == "":
        #     raise ValueError(f"Must specify entity_name")"

Reply:
my suggestion/ideal would be to
1) remind the user to log in before running the demo and 
2) drop all the many references to entity, and mention it's optional in a note somewhere

comment about API key + source code seem to be in conflict, so I want to make sure this would be ok!

Reply:
We have been flip flopping on this. Shawn suggested we basically force that that user specifies all 3 parts always. No reading from netrc/other settings. I will probably reinforce this before the release